derive_builder/tests/compile-fail/builder_setter_attrs.rs

```diff
@@ -0,0 +1,17 @@
+#![deny(unused_must_use)]
+
+#[macro_use]
+extern crate derive_builder;
+
+#[derive(Debug, PartialEq, Default, Builder, Clone)]
+pub struct Lorem {
+    ok: String,
+
+    // This foolish repr() attribute generates an unused attribute warning
```

Comment:
What's `repr` referring to here?

Reply:
This was leftover from an earlier version of this test case.